### ISSUE TITLE
Rejuvenate home-assistant-sun-card

### ISSUE THREAD

**Issue by ThomDietrich** [rejuvenate/rejuvenate#13]
The project has an active user base and a bigger set of open PRs. The maintainer recently archived all of his repositories.

I propose to [_rejuvenate_](https://github.com/rejuvenate/rejuvenate/blob/master/README.md) the repository and will create and set up the fork today or tomorrow. Which of the existing PR holders and recent code developers (I ignored the translation providers) would be interested to step up as one of the maintainers? Possible candidates: @ThomDietrich @avataar @Pjanssen @JosefRypacek @cerebrate @selfish

- Repo: https://github.com/AitorDB/home-assistant-sun-card
- PRs: https://github.com/AitorDB/home-assistant-sun-card/pulls
- Network: https://github.com/AitorDB/home-assistant-sun-card/network

**Comment by selfish** [comment]
@ThomDietrich well done on the initiative!

I just forked it myself, under the name 'lovelace-sun-card' intending to keep it alive and I already made some cleanup, I setup CI with auto-release, and made other improvements. I don't mind not owning the new repo, and I'm happy to contribute.

With that said, I think it's a good idea to reconsider some of the decisions, especially the name that doesn't conform with HACS recommendations.

Wdyt?

TL;DR: I'm happy to join as a maintainer, and I have lots of ideas for improvement.

**Comment by avataar** [comment]
I'd be happy to invest some time too. My fork has a minor fix to the develop branch that made it actually build and work for me (as well as a Bulgarian translation and README cleanup): https://github.com/avataar/home-assistant-sun-card/tree/develop.

I don't care who owns the repo either. Let's bring this awesome thing back to life!

**Comment by ThomDietrich** [comment]
Hey both, thanks for the feedback!

Great to hear you already started a bit of a clean up @selfish. It doesn't necessarily matter where the repository resides, as long as it is alive. I found that multiple maintainers and an organization can sometimes ensure that better and for longer. I will create the fork now and make the two of you maintainers. Let's discuss any further ideas over there!

Cheers!

https://github.com/rejuvenate/sun-card

**Comment by selfish** [comment]
Invite accepted. We can continue to discuss over there.
Thanks!

**Comment by selfish** [comment]
Hi @ThomDietrich,

I am writing here to avoid cluttering of the sun-card repo with off-topic matters.

I want to begin by expressing my gratitude for the opportunity to contribute to rejuvenate/sun-card project. 

However, after some thought and consideration, I have decided to pursue my own fork of the project. I believe it will allow me to better explore and develop the ideas I have in mind and make the most of my skill set.

As I step down as a maintainer, I want you to know that you and the rest of the team are welcome to use any of the work I have contributed to the project thus far. My goal is to foster a spirit of collaboration and continued growth within the open-source community.

I wish the project and all its contributors the very best in their future endeavors. Thank you once again for the opportunity to be part of this project, and I look forward to seeing how it continues to evolve.

**Comment by avataar** [comment]
Hi @selfish,

Sorry to see you go. I also hope we can continue to share ideas and, if needed, help each other.

Good luck with your fork!
